src/packageurl/contrib/django/models.py

```diff
@@ -32,13 +32,17 @@
 
 from packageurl import PackageURL
 from packageurl.contrib.django.utils import purl_to_lookups
+from packageurl.contrib.django.utils import without_empty_values
 
 warnings.warn(
     "`purl_to_lookups`, `without_empty_values` is moved from packageurl.contrib.django.models"
     " to packageurl.contrib.django.utils",
     DeprecationWarning,
 )
 
+purl_to_lookups = purl_to_lookups
+without_empty_values = without_empty_values
```

Comment:
Those statements are useless.

Reply:
Yeah I messed up that commit, made some changes in a new commit